tests/Makefile

```diff
@@ -132,7 +132,8 @@ DIRS += fsflags
 
 ifdef MYST_NIGHTLY_TEST
 DIRS += glibc
-DIRS += aspnetcore
+DIRS += aspnetcore5
+DIRS += aspnetcore6
```

Comment:
I think we should put both aspnetcore5 and aspnetcore6 in the PR pipeline as they are taking much shorter time than before.

Reply:
Running these two tests take about 20-30 minutes, and the [PR pipeline timed out](https://openenclave.visualstudio.com/ACC-Services/_build/results?buildId=13161&view=logs&j=ab477ad6-9781-5b54-82ca-a0141b26726e&t=a1e294d4-8895-5a45-41ff-022cede0621b). Should move back to nightly pipeline

Reply:
Sure